mmcv/runner/hooks/lr_updater.py

```diff
@@ -118,7 +118,8 @@ def before_run(self, runner):
                 self.base_lr.update({k: _base_lr})
         else:
             for group in runner.optimizer.param_groups:
-                group.setdefault('initial_lr', group['lr'])
+                # group.setdefault('initial_lr', group['lr'])
+                group['initial_lr'] = group['lr']
```

Comment:
When resuming from a checkpoint, if 'initial_lr' is not saved, it will be set according to the optimizer params. So if we directly set the `initial_lr` as `lr`, I think it may not be an expected behavior.